common/supplemental/likelySubtags.xml

```diff
@@ -2166,14 +2160,10 @@ not be patched by hand, as any changes made in that fashion may be lost.
 		<!--{ Sicilian; ?; ? } => { Sicilian; Latin; Italy }-->
 		<likelySubtag from="sco" to="sco_Latn_GB"/>
 		<!--{ Scots; ?; ? } => { Scots; Latin; United Kingdom }-->
-		<likelySubtag from="scs" to="scs_Latn_CA"/>
-		<!--{ North Slavey; ?; ? } => { North Slavey; Latin; Canada }-->
 		<likelySubtag from="sd" to="sd_Arab_PK"/>
 		<!--{ Sindhi; ?; ? } => { Sindhi; Arabic; Pakistan }-->
 		<likelySubtag from="sd_Deva" to="sd_Deva_IN"/>
 		<!--{ Sindhi; Devanagari; ? } => { Sindhi; Devanagari; India }-->
-		<likelySubtag from="sd_IN" to="sd_Deva_IN"/>
-		<!--{ Sindhi; ?; India } => { Sindhi; Devanagari; India }-->
 		<likelySubtag from="sd_Khoj" to="sd_Khoj_IN"/>
```

Comment:
This was specifically added for a ticket, https://unicode-org.atlassian.net/browse/CLDR-15347. I thought I added the necessary data to get it auto-generated but perhaps not.

Reply:
@pedberg-icu  I've reverted this line. I couldn't trace from the debug lines why this wasn't getting shown. If anything, there was a potential candidate `sd_IN` -> `sd_Arab_IN` which is not what you requested.

Reply:
reversion here https://github.com/unicode-org/cldr/compare/unicode-org:2194483..unicode-org:f18db5b

Reply:
Thanks! Approved.